website/content/en/applications.adoc

```diff
@@ -3,41 +3,57 @@ title: "About Applications for FreeBSD"
 sidenav: about
 ---
 
-= About Applications for FreeBSD
+= Applications for FreeBSD
 
-== Experience the possibilities with FreeBSD
+FreeBSD goes beyond what's commonly expected of a UNIX(R) workstation.
 
-FreeBSD can handle nearly any task you would expect of a UNIX(R) workstation, as well as many you might not expect:
+== Thousands of applications
 
-== FreeBSD is a true open system with full source code.
+There are more than thirty thousand link:../ports/[ports] and related packages.
```

Comment:
Fourty? Fifty? Freshports says 57000 and a bit right now.

Reply:
<https://www.freebsd.org/status/report-2022-07-2022-09/#_ports_collection>: 

> … just over 30,500 …

Prior discussion (in GitHub commentary, maybe) explained how the total is calculated for purposes such as this.